### ISSUE TITLE
wchar_t disappears when C++ interop is enabled

### ISSUE THREAD

**Issue by grynspan** [apple/swift#62792]
If you enable C++ interop, you can no longer use `wchar_t` in Swift. `wchar_t` is rarely used on Darwin/Linux, but is necessary for Win32.

The type itself may be a typedef or a builtin type in C. `CWideChar` is defined in the Swift standard library but is not mapped to `wchar_t` in the importer (https://github.com/apple/swift/issues/55616) and is the wrong width on Win32 anyway (https://github.com/apple/swift/issues/55617).

**Comment by grynspan** [comment]
@compnerd Did you fix this? :)

**Comment by compnerd** [comment]
Yeah, https://github.com/swiftlang/swift/pull/68397 should fix this.
